src/path_rewriting.rs

```diff
@@ -40,6 +40,16 @@ pub fn canonicalize_path<P: AsRef<Path>>(path: P) -> io::Result<PathBuf> {
     Ok(path)
 }
 
+fn normalize_path<P: AsRef<Path>>(path: P) -> PathBuf {
```

Comment:
Isn't there a Rust builtin method to do something like this?

Reply:
There is canonicalize but you need to have a "true" path.
I checked the doc but didn't find anything except the components() trick.

Reply:
Yeah, we call canonicalize a few lines earlier, but it fails when the path is not there (which can be the case if you don't have a local clone of the repo).